README.md

```diff
@@ -84,8 +84,14 @@ make TAGS=nolzma
 
 ### Steps
 
-To install Mender on a device from source, please run the following commands
-inside the cloned repository:
+To install Mender on a device from source, first clone the repository in the correct folder
+structure inside your GOPATH (typically `$HOME/go`):
+
+```
+git clone https://github.com/mendersoftware/mender.git $HOME/go/src/github.com/mendersoftware/mender
```

Comment:
Maybe use $GOPATH instead of $HOME/go then?

Reply:
:+1: